src/partials/title.html

```diff
@@ -0,0 +1,17 @@
+<article class="o-sg-section o-sg-section--padded o-sg-section__atomic-category">
```

Comment:
hmh, these are all related to the visual app, right?
If so my we move them inside the `arc/app` folder

Reply:
Yes, but it's for styleguide visuals only. I just moved them out of javascript.

Reply:
done. moved into app.